src/web/ViewBase.tsx

```diff
@@ -113,32 +113,32 @@ export abstract class ViewBase<P extends Types.ViewProps, S> extends RX.ViewBase
             return SyncTasks.Resolved<void>();
         }
 
-        const container = this._getContainer();
-        if (!container) {
-            return SyncTasks.Resolved<void>();
-        }
+        const deferred = SyncTasks.Defer<void>();
+        ViewBase._reportLayoutChange(() => {
```

Comment:
I worry about the perf implications of this change. The previous code tried very hard to avoid creating timers in the frequent case where the view size and position didn't change. The new code always allocates a timer on every update. Have you done sufficient perf measurements on parts of the code that will be impacted by this (e.g. the VMV in Skype)?

Reply:
I see. I wonder if there's an easy way to use the new behavior only if we know that it's going to help. Maybe only if a popup is being rendered for the first time. I'll report back shortly, thanks!